instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/InstrumenterBuilder.java

```diff
@@ -129,6 +129,11 @@
         InstrumenterConstructor.propagatingToDownstream(setter), SpanKindExtractor.alwaysClient());
   }
 
+  /** Returns a new {@link Instrumenter} which will create client spans. */
+  public Instrumenter<REQUEST, RESPONSE> newClientInstrumenter() {
```

Comment:
I think `newInstrumenter(SpanKindExtractor)` is fine for the rare case of an instrumenter that only generates client spans without propagation, or otherwise the semantics of the overloads for `newClientInstrumenter` become too different

Reply:
Excellent point, removed.